app/helpers/forever_style_guide/application_helper.rb

```diff
@@ -257,6 +257,9 @@ def tos_url
     def privacy_url
       www_url('/privacy')
     end
+    def careers_url
```

Comment:
Missing return, and as mentioned on the other branch, I'd expect this to work like the social links